libsolidity/parsing/Parser.cpp

```diff
@@ -234,6 +235,7 @@ ASTPointer<ImportDirective> Parser::parseImportDirective()
 		if (m_scanner->currentToken() == Token::As)
```

Comment:
Shouldn't the `else` case clear `unitAliasLocation`? This case is `import "abc.sol"`. There is no name being declared for that module, so there should be no location.

Reply:
Actually setting it to an empty location at its definition would probably fix this.

Reply:
yes, thx.